sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/UnsafeRow.java

```diff
@@ -241,14 +245,37 @@ public void setFloat(int ordinal, float value) {
   @Override
   public void setDecimal(int ordinal, Decimal value, int precision) {
```

Comment:
can you update InternalRow and UnsafeRow's documentation to say in order to set null on a decimal field, the caller must use setDecimal(ordinal, null, ...) instead of setNull?